packages/adaptive_scaffold/lib/src/adaptive_scaffold.dart

```diff
@@ -0,0 +1,462 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/material.dart';
+import 'adaptive_layout.dart';
+import 'breakpoint.dart';
+import 'breakpoints.dart';
+import 'slot_layout.dart';
+import 'slot_layout_config.dart';
+
+/// [AdaptiveScaffold] is an abstraction that passes properties to
+/// [AdaptiveLayout] and reduces repetition and a burden on the developer.
+class AdaptiveScaffold extends StatefulWidget {
```

Comment:
Again, the API docs here should be more verbose. Include links and "See also" sections for things that are involved, define terms, etc.

This comment is the landing page for the main class in the package.